classes/local/item_type_list.php

```diff
@@ -97,13 +97,14 @@ public static function get_default_types() {
             new fields\course_groups_learner_field(),
             new fields\idnumber_learner_field(),
             new fields\email_learner_field(),
-            new measures\assignment_resubmit_count(),
+            new fields\learner_profile_field(),
+            //new measures\assignment_resubmit_count(),
             new measures\assignment_status(),
             new measures\attendance_sessions_summary(),
-            new measures\checklist_complete(),
+            //new measures\checklist_complete(),
             new measures\course_grade(),
             new measures\course_section_activity_completion(),
-            new measures\grade_category_activity_completion(),
+            //new measures\grade_category_activity_completion(),
```

Comment:
Why have these measures been commented out?

Reply:
Forgot about those. I will restore/uncomment. This is possibly something for the future. Create a plugin setting/config var to store enabled measures. LW doesn't use those measures, sometimes I click on Assignment resubmit count rather than Assignment status. Needs a UI Improvement.